federation/pkg/kubefed/util/util.go

```diff
@@ -64,17 +68,30 @@ func (a *adminConfig) PathOptions() *clientcmd.PathOptions {
 	return a.pathOptions
 }
 
-func (a *adminConfig) HostFactory(host, kubeconfigPath string) cmdutil.Factory {
+func (a *adminConfig) FedClientSet(context, kubeconfigPath string) (*fedclient.Clientset, error) {
```

Comment:
Call this function `FederationClientset`. Expand fed and lower s in set. It is better for readability and Clientset follows the convention elsewhere.